pkg/cli/admin/release/git.go

```diff
@@ -205,15 +207,26 @@ func mergeLogForRepo(g *git, repo string, from, to string) ([]MergeCommit, error
 		}
 
 		msg := records[3]
+		msg = rePrefix.ReplaceAllString(strings.TrimSpace(msg), "")
 		if m := reBug.FindStringSubmatch(msg); m != nil {
 			mergeCommit.Subject = msg[len(m[0]):]
-			mergeCommit.Bug, err = strconv.Atoi(m[1])
-			if err != nil {
-				return nil, fmt.Errorf("could not extract bug number from %q: %v", msg, err)
+			for _, part := range strings.Split(m[1], ",") {
+				for _, subpart := range strings.Split(part, " ") {
+					subpart = strings.TrimSpace(subpart)
+					if len(subpart) == 0 {
+						continue
+					}
+					bug, err := strconv.Atoi(subpart)
+					if err != nil {
+						continue
```

Comment:
log the error at least at high level?

Reply:
Technically this is impossible with the current regex.